scripts/README.md

```diff
@@ -229,6 +229,62 @@ kubectl get pods || fail_test
 success
 ```
 
+## Using the `performance-tests.sh` helper script
+
+This is a helper script for Knative performance test scripts. In combination
+with specific Prow jobs, it can automatically manage the environment for running
+benchmarking jobs for each repo. To use it:
+
+1. Source the script.
+
+1. [optional] Customize GCP project settings for the benchmarks. Set the
+   following environment variables if the default value doesn't fit your needs:
+
+   - `PROJECT_NAME`: GCP project name for keeping the clusters that run the
+     benchmarks. Defaults to `knative-performance`.
+   - `SERVICE_ACCOUNT_NAME`: Service account name for controlling GKE clusters
+     and interacting with [Mako](https://github.com/google/mako) server. It MUST have
+     `Kubernetes Engine Admin` and `Storage Admin` role, and be
+     [whitelisted](https://github.com/google/mako/blob/master/docs/ACCESS.md) by
+     Mako admin. Defaults to `mako-job`.
+
+1. [optional] Customize path of the benchmarks. This root folder should
+   contain and only contain all benchmarks you want to run continuously.
+   Set the following environment variable if the default value doesn't fit your
+   needs:
+
+   - `BENCHMARK_ROOT_PATH`: Benchmark root path, defaults to `test/performance/benchmarks`.
```

Comment:
what does overriding this do?

Reply:
Each repo can decide which folder to put its benchmarks (each benchmark is a subfolder), and override this environment variable to be the path of that folder.
Does it answer your question?

Reply:
yeah that's pretty much what I guessed, can you add a description here?

Reply:
Done.